model/lifecycle.go

```diff
@@ -807,7 +807,7 @@ func createTasksForBuild(ctx context.Context, creationInfo TaskCreationInfo) (ta
 		}
 
 		// update existing exec tasks
-		grip.Error(message.WrapError(task.AddDisplayTaskIdToExecTasks(ctx, id, execTasksThatNeedParentId), message.Fields{
+		grip.Error(message.WrapError(task.AddDisplayTaskInfoToExecTasks(ctx, id, dt.Name, execTasksThatNeedParentId), message.Fields{
```

Comment:
Maybe I'm missing something about how task creation works, but isn't this called before the exec task is in the DB? Does this actually set the display task info on all task docs when they're inserted in the DB? `createTasksForBuild` doesn't insert any tasks itself, that's done in a transaction that the caller is expected to do themselves, so I would have expected this line to not add any display task info to tasks that are being created here.

Any reason not to just set the display task display name in `createOneTask`? That seems more straightforward than doing it in a separate update.